specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/CommonDefinitions.json

```
@@ -3674,6 +3674,7 @@
         },
         "instanceCount": {
           "type": "number",
```

Please make this change here and in the other three locations.
```suggestion
          "type": "integer",
```

---

Thank you, @mikekistler, for the review. Fixed.